homeassistant/components/home_connect/sensor.py

```diff
@@ -80,10 +80,10 @@ async def async_update(self) -> None:
             else:
                 self._state = status[self._key].get(ATTR_VALUE)
                 if self._key == BSH_OPERATION_STATE:
-                    # Value comes back as an enum, we only really care about the
-                    # last part, so split it off
-                    # https://developer.home-connect.com/docs/status/operation_state
-                    self._state = self._state.split(".")[-1]
+                    self._state = (
+                        OPERATION_STATE_TRANSLATION.get(self._state)
+                        or self._state.split(".")[-1]
+                    )
```

Comment:
Let's simplify this change somewhat. Just lowercase the value instead. That seem to be the only main requirements on on keys for translations.

Reply:
Changed it! Thanks!